home/tests/test_api.py

```diff
@@ -460,6 +594,115 @@ def test_detail_view_no_content_page(self, uclient):
         assert content == {"page": ["Page matching query does not exist."]}
         assert content.get("page") == ["Page matching query does not exist."]
 
+    def test_wa_image(self, uclient):
+        """
+        Test that API returns image ID for whatsapp
+        """
+        page = self.create_content_page(body_type="WhatsApp", image=True)
+        response = uclient.get(f"/api/v2/pages/{page.id}/?whatsapp=true")
+        content = response.json()
+
+        image_id = content["body"]["text"]["value"]["image"]
+        assert image_id == page.whatsapp_body._raw_data[0]["value"]["image"]
+
+    def test_messenger_image(self, uclient):
+        """
+        Test that API returns image ID for messenger
+        """
+        page = self.create_content_page(body_type="Messenger", image=True)
+        response = uclient.get(f"/api/v2/pages/{page.id}/?messenger=true")
+        content = response.json()
+
+        image_id = content["body"]["text"]["image"]
+        assert image_id == page.messenger_body._raw_data[0]["value"]["image"]
+
+    def test_viber_image(self, uclient):
+        """
+        Test that API returns image ID for viber
+        """
+        page = self.create_content_page(body_type="Viber", image=True)
+        response = uclient.get(f"/api/v2/pages/{page.id}/?viber=true")
+        content = response.json()
+
+        image_id = content["body"]["text"]["image"]
+        assert image_id == page.viber_body._raw_data[0]["value"]["image"]
+
+    def test_wa_media(self, uclient):
+        """
+        Test that API returns media ID for whatsapp
+        """
+        page = self.create_content_page(media=True)
+        response = uclient.get(f"/api/v2/pages/{page.id}/?whatsapp=true")
+        content = response.json()
+
+        media_id = content["body"]["text"]["value"]["media"]
+        assert media_id == page.whatsapp_body._raw_data[0]["value"]["media"]
+
+    def test_wa_doc(self, uclient):
+        """
+        Test that API returns doc ID for whatsapp
+        """
+        page = self.create_content_page(doc=True)
+        response = uclient.get(f"/api/v2/pages/{page.id}/?whatsapp=true")
+        content = response.json()
+
+        doc_id = content["body"]["text"]["value"]["document"]
+        assert doc_id == page.whatsapp_body._raw_data[0]["value"]["document"]
+
+    def test_wa_messages_only_1_image(self, uclient):
+        """
+        Test that when many images are assigned to messages in the test, only 1 image is created in the test
+        """
```

Comment:
Having tests for our local test helpers may be an indication that they're too complex and we should rather be more explicit in the tests themselves.